operators/churner/cmd/Dockerfile

```diff
@@ -13,14 +13,14 @@ COPY encoding /app/encoding
 COPY go.mod /app
 COPY go.sum /app
 
-WORKDIR /app/churner
+WORKDIR /app/operators/churner
 
 RUN --mount=type=cache,target=/go/pkg/mod \
   --mount=type=cache,target=/root/.cache/go-build \
   go build -o ./bin/churner ./cmd
```

Comment:
And here build it from `./churner/cmd` (rather than `./cmd`)

And for ejector image, this will be the only difference to pay attention to (`./ejector/cmd`)